app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonDatabaseAdapter.java

```diff
@@ -314,31 +315,31 @@ private void initializeContacts() throws IOException {
     private void initializeDatabase() throws IOException {
         final File file = new File(this.profilesPath);
         if (!file.exists()) {
-            LOGGER.warn("Part of the database didn't exist: %s", this.profilesPath);
-            final Writer writer = this.fileSystem.getWriter(this.profilesPath);
-            writer.write("[]");
-            writer.close();
-            LOGGER.info("Created the part of the database: %s", this.profilesPath);
+            LOGGER.warn("Part of the database didn't exist: {}", this.profilesPath);
+            try (Writer writer = this.fileSystem.getWriter(this.profilesPath)) {
+                writer.write("[]");
+                writer.close();
+                LOGGER.info("Created the part of the database: {}", this.profilesPath);
+            }
         }
     }
 
     /**
      * Deserialize the instance using readObject to ensure invariants and security.
+     *
      * @param stream The serialized object to be deserialized
      */
     private void readObject(final ObjectInputStream stream) throws IOException, ClassNotFoundException {
         stream.defaultReadObject();
-        ensureClassInvariant();
+        this.ensureClassInvariant();
     }
 
     /**
      * Ensure that the instance meets its class invariant
-     * @throws InvalidObjectException Thrown when the state of the class is unstable
      */
-    private void ensureClassInvariant() throws InvalidObjectException {
-        assertNotNull(this.contactsPath);
-        assertNotNull(this.profilesPath);
-        assertNotNull(this.fileSystem);
+    private void ensureClassInvariant() {
+        assertThat(this.contactsPath, notNullValue());
+        assertThat(this.profilesPath, notNullValue());
+        assertThat(this.fileSystem, notNullValue());
```

Comment:
Well, ok, in this case it doesn't really matter. Just a personal preference.
However, there are a lot more reasons to prefer assertThat over the old JUnit functions, see https://objectpartners.com/2013/09/18/the-benefits-of-using-assertthat-over-other-assert-methods-in-unit-tests/